source/MRViewer/MRViewer.h

```diff
@@ -439,6 +439,7 @@ class MRVIEWER_CLASS Viewer
     MouseController mouseController;
     TouchesController touchesController;
     SpaceMouseController spaceMouseController;
+    std::shared_ptr<SpaceMouseHandler> spaceMouseHandler;
```

Comment:
mb better have get accessor, not to allow user reset this pointer

Reply:
fixed